src/main/kotlin/no/nav/sykdig/nasjonal/api/NasjonalOppgaveDataFetcher.kt

```diff
@@ -48,5 +42,22 @@ class NasjonalOppgaveDataFetcher(
         return null
```

Comment:
Tenker det er fint med ein FINNES_IKKE her også.
```suggestion
        eturn NasjonalOppgaveStatus(oppgaveId, NasjonalOppgaveStatusEnum.FINNES_IKKE)
```